HISTORY.rst

```diff
@@ -2,6 +2,27 @@
 History
 =======
 
+3.55.0 (2022-03-24)
+-------------------
+
+Changes
+
+- Tweaked Oauth App org restriction docs. Alters the trigger for warn_oauth_restricted error handler
```

Comment:
Docs changes shouldn't go in release notes.

With respect to the other part of the PR: what is the user-facing change? I don't know the relevance of  the `warn_oauth_restricted` trigger error handler to me as an end-user.